Makefile

```diff
@@ -43,6 +44,10 @@ checkformatting: python
 .PHONY: test
 test: backend-test frontend-test
 
+.PHONY: functests
+functests: python
+	@tox -qe functests
```

Comment:
We should add `functests` to `make sure` as well

Reply:
Done